package.json

```diff
@@ -44,10 +46,13 @@
     "ember-cli-babel": "^7.26.11",
     "ember-cli-htmlbars": "^6.1.1",
     "ember-fetch": "^8.1.2",
-    "ember-intl": "^5.7.2",
-    "ember-truth-helpers": "^3.1.1",
+    "ember-intl": "^6.0.0",
+    "ember-truth-helpers": "^4.0.3",
```

Comment:
We _could_ go for something like `"^3.1.1 || ^4.0.3"` but I think it makes sense to do a hard cut here. Especially since it's easy to update from 3 to 4.